frontend/nuxt.config.js

```diff
@@ -48,8 +49,8 @@ export default {
             { name: 'twitter:description', content: SITE_DESCRIPTION },
             { name: 'twitter:image', content: PREVIEW_IMAGE },
         ],
-        link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.ico' }],
-        script: [{ src: '/js/HeapIO.js' }],
+        link: [{ rel: 'icon', type: 'image/x-icon', href: path.join(__dirname, 'dist/favicon.ico') }],
+        script: [{ src: path.join(__dirname, 'dist/js/HeapIO.js') }],
```

Comment:
I think `__dirname` is actually deprecated by node and you should use [`import.meta.url` instead](https://flaviocopes.com/fix-dirname-not-defined-es-module-scope/)

But why do you actually need to do that?

Reply:
Please remove those changes and merge logic from #575

Reply:
Please remove those changes and merge logic from #575. Please note that you will need to provide correct env var for the npm run build to activate correct router mode

Reply:
Where do you provide correct `ENABLE_FILE_PROTOCOL` env variable to the frontend build command? Or do you expect the developer to change it in the env file?